.github/workflows/test:e2e.yml

```diff
@@ -77,17 +75,35 @@ jobs:
         uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run kurtosis
-        run: kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+        run: kurtosis run --enclave pp --args-file ${KURTOSIS_ARG_FILE} --image-download always .
```

Comment:
@Freyskeyd For some reason on Ubuntu 22.04 I had an issue that kurtosis ignores changes in the `fork12-pessimistic.yml` (it always takes default `agglayer_image` specified in the `input_parser.star`  so I had to pass the changes manually, e.g.:
```
kurtosis run --enclave cdk --args-file .github/tests/fork12-pessimistic.yml --image-download always . '{"args": {"agglayer_prover_sp1_key": "0xykey", "additional_services":["prometheus_grafana", "tx_spammer"], "agglayer_image":"agglayer-local"}}'
```